apps/extension/src/routes/popup/settings/settings-clear-cache.tsx

```diff
@@ -26,7 +27,14 @@ export const SettingsClearCache = () => {
               won’t be lost!
             </p>
           </div>
-          <Button variant='gradient' size='lg' className='w-full'>
+          <Button
+            variant='gradient'
+            size='lg'
+            className='w-full'
+            onClick={() => {
+              void (async () => await swClient.clearCache())();
+            }}
+          >
```

Comment:
`onClick={void swClient.clearCache}`

Reply:
it doesn't work!on click do nothing

Reply:
` onClick={() => void swClient.clearCache()}` works